src/usyscalls.s

```diff
@@ -0,0 +1,36 @@
+.include "src/machine-word.inc"
+
+### User payload = code + readonly data for U-mode ############################
+#
+.macro  macro_syscall nr
+        li      a7, \nr                 # for fun let's pretend syscall is kinda like Linux: syscall nr in a7, other arguments in a0..a6
+        ecall
+.endm
+.macro  macro_sys_poweroff exit_code
```

Comment:
macro_sys_poweroff, macro_sys_print - I don't think these are used anymore. Should be safe to remove them.